### DISCUSSION TITLE
<title>Images used in README files do not load.

### DISCUSSION THREAD

**Discussion by rahulrao0209** [community/community#45727] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,
The images that are being used in the README file of my repository on GitHub do not load. I am using markdown for the README.

Attaching links to the repositories pertaining to this issue.
https://github.com/rahulrao0209/checkpass
https://github.com/rahulrao0209/clipboard

Kindly advise regarding a fix for the same. Thank You!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @rahulrao0209 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by rahulrao0209** [reply]
Thank You!
